client/postit/src/App.js

```diff
@@ -4,8 +4,11 @@ import SignUp from "./components/SignUp";
 import SignIn from "./components/SignIn";
 import API_ROUTES from "./utils/constants";
 import { useState } from "react";
+import { usePromiseTracker } from "react-promise-tracker";
+import { RotatingLines } from "react-loader-spinner";
```

Comment:
This too can be avoided, try as much as possible not to add unnecessary dependencies to your project, they can become a pain in the future especially when they are no longer maintained. Here you could use normal css.Css loaders exist..

Reply:
thank you